src/main/java/com/blackducksoftware/integration/hub/HubIntRestService.java

```diff
@@ -375,15 +253,15 @@ public ProjectItem getProject(final String projectUrl) throws IOException, BDRes
 
 	}
 
-	public ReleaseItem getProjectVersion(final String versionUrl)
-			throws IOException, BDRestException, URISyntaxException {
-		final ClientResource resource = createClientResource(versionUrl);
+	public ReleaseItem getProjectVersion(final String versionUrl) throws IOException, BDRestException,
+	URISyntaxException {
+		final ClientResource resource = getRestConnection().createClientResource(versionUrl);
```

Comment:
We do a number of GET method calls.  I'm wondering if this should be encapsulated in RestConnection in a 
public ClientResource submitGETRequest(url) {
final ClientResource resource = restConnection.createClientResource(url);
		resource.setMethod(Method.GET);
		restConnection.handleRequest(resource);
 return resource;
}
we could add methods for PUT,POST, and DELETE as well to cover the verbs for REST calls.  Since we have the same pattern multiple times I was wondering if it makes sense to encapsulate it or because we may need ClientResource and RestConnection it is better to leave it the way it is and call the three lines of code in multiple places.